docs/development/pigs/pig-011.rst

```diff
@@ -0,0 +1,138 @@
+.. include:: ../../references.txt
+
+.. _pig-011:
+
+**********************************************
+PIG 11 - Gammapy dependencies and distribution
+**********************************************
+
+* Author: Christoph Deil, Axel Donath, Régis Terrier
+* Created: June 6, 2019
+* Accepted: tbd
+* Status: draft
+* Discussion: `GH 2218`_
+
+Abstract
+========
+
+As preparation for the planned Gammapy v1.0 release this fall, we propose to
+clarify which Gammapy dependencies are required and which ones are optional. We
+also propose to increase the minimum required version for some dependencies.
+Finally, we propose some updates to the Gammapy supported distribution channels
+and installation instructions.
+
+This represents that plan for 2019 to be implemented before v1.0. Things are
+likely to evolve in the coming years, as Gammapy supports different projects,
+especially CTA.
+
+One reason to write a PIG for this (as opposed to just making these decisions
+among the Gammapy lead developers or coordination committee) is to create an
+opportunity for user feedback. Let us know what you think!
+
+Introduction
+============
+
+Dependencies
+============
+
+- list which dependencies are required and optional?
+- 
+
+Versions
+--------
+
+Drop Python 3.5 support
+-----------------------
+
+As already mentioned in `pig-003`_, we now suggest to drop support for Python
+3.5, already in the next release, Gammapy v0.13 in July 2019, for the following
+reasons:
+
+- We're not aware of any users that have to or even want to use Python 3.5, most
+people use Gammapy on Python 3.6 or 3.7 now via conda.
+- Reduce maintenance effort. conda-forge now only builds binaries for Python 3.6
```

Comment:
We have e.g. regions as a dependency that has C extension and doesn't have binary wheels, and soon astropy-healpix. So at the moment using `pip` to install Gammapy dependencies is not great, it will run a few minutes to compile things. (might just be 1-2 minutes).